Documentation/install-metering.md

```diff
@@ -42,7 +42,7 @@ There are multiple installation methods depending on your Kubernetes platform an
 
 Using OLM is the recommended option as it ensures you are getting a stable release that we have tested.
 
-OLM is currently only supported on Openshift 4.x.
+OLM is currently only supported on Openshift 4.2 and newer.
```

Comment:
nit:
Openshift 4.2+ 
or
newer versions?

Reply:
is the word "and" the problem? What about `Openshift 4.2 or newer`?

Reply:
I just thought it read weirdly, but only changing "and" to "or" would be an improvement imo.